pkg/storaged/format-dialog.jsx

```diff
@@ -330,6 +330,7 @@ function format_dialog_internal(client, path, start, size, enable_dos_extended,
                                                     return _("Passphrase cannot be empty");
                                             },
                                             visible: vals => is_encrypted(vals) && vals.crypto != " keep",
+                                            new_passphrase: true
```

Comment:
ITYM `new_password` here, no?

Reply:
OMG! Good catch, this should be `new_password`.  Thanks.